Updating build index: /home/vsts/conda/conda-bld

TEST START: /tmp/tmpbn8t25ou/linux-64/pytest-marks-0.4-py36_0.tar.bz2
Adding in variants from internal_defaults
Reloading output folder: /home/vsts/conda/conda-bld
Mamba failed to solve:
 - pytest-marks 0.4.* py39_0
 - python
 - setuptools
 - pytest >2.0.2
 - python

with channels:
 - file:///tmp/tmpbn8t25ou

The reported errors are:
   Encountered problems while solving:
     - nothing provides requested pytest-marks 0.4.* py39_0
   

Leaving build/test directories:
  Work:
 /home/vsts/conda/conda-bld/work 
  Test:
 /home/vsts/conda/conda-bld/test_tmp 
Leaving build/test environments:
  Test:
source activate  /home/vsts/conda/conda-bld/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh 
  Build:
source activate  /home/vsts/conda/conda-bld/_build_env 


